exercises/practice/high-scores/.meta/example.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <array>
+#include <vector>
+
+using namespace std;
```

Comment:
This is the only think that I really don't like. Please, no `using` directive in the global namespace of `.h` files, it forces other files that `#include` this to "use" that namespace themselves. Personally I'd use the prefix `std::` everywhere in the `.h` file or add a `using std::vector` declaration in the `namespace arcade` but if you really want the `using namespace` you could move it into the `namespace arcade`.

Reply:
Yes, I am completely with you there.
I have not used it there yet and was unsure of that. I am glad you don't like it.